js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/useCreateIndex.ts

```diff
@@ -36,8 +37,11 @@ export const useCreateIndex = <TValues extends unknown>() => {
         try {
           result = await getApiRouteForCurrentDB().post(
             `index`,
-            values,
-            `collection=${collectionName}`,
+            {
+              ...values,
+              name: (values.name as string)?.normalize() || undefined
```

Comment:
```suggestion
              name: (String(values.name))?.normalize() || undefined
```

Considerable alternative to asserting it..